### ISSUE TITLE
Delay warmup cluster task until DNS is updated

### ISSUE THREAD

**Issue by deepthidevaki** [zeebe-io/zeebe-cluster-testbench#226]
In the workflow "prepare-zeebe-cluster-in-camunda-cloud", the task "Warmup cluster" always fails because it is executed before the DNS entry for the newly created cluster is updated. The workflow can proceed only after resolving the incident, and this happens all the time. To prevent this we can add a timer before the warmup task. Not sure how much delay should be added, but assume a few minutes would be good.

**Comment by npepinpe** [comment]
Duplicate of #201
